packages/fxa-auth-server/test/scripts/remove-accounts.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+import {
+  retreiveUnverifiedAccounts,
+  cancelSubscriptionsAndDeleteCustomer,
+  issueRefund,
+} from '../../scripts/remove-accounts';
+
+const ROOT_DIR = '../..';
+const cp = require('child_process');
+const util = require('util');
+const path = require('path');
+const execAsync = util.promisify(cp.exec);
+const sinon = require('sinon');
+const { assert } = require('chai');
+
+const cwd = path.resolve(__dirname, ROOT_DIR);
+const execOptions = {
+  cwd,
+  env: {
+    ...process.env,
+    NODE_ENV: 'dev',
+    LOG_LEVEL: 'error',
+    AUTH_FIRESTORE_EMULATOR_HOST: 'localhost:9090',
+  },
+};
+
+describe('scripts/remove-accounts startup', () => {
+  it('does not fail', function () {
+    this.timeout(20000);
+    return execAsync(
+      'node -r esbuild-register scripts/remove-accounts.ts',
+      execOptions
+    );
+  });
+});
+
+describe('scripts/remove-accounts - retreiveUnverifiedAccounts', () => {
+  it('filters for unverified accounts that are older than 16 days old', async () => {
+    const accounts = [
+      {
+        uid: '1234',
+        createdAt: new Date(),
+        email: 'user1@test.com',
+        emailCode: undefined,
+        normalizedEmail: 'user1@test.com',
+        emailVerified: false,
+        verifierVersion: 1,
+        verifyHash: undefined,
+        authSalt: undefined,
+        kA: undefined,
+        wrapWrapKb: undefined,
+        verifierSetAt: 0,
+        locale: 'en-US',
+      },
+      {
+        uid: '1234',
+        createdAt: new Date().getDate() - 17,
```

Comment:
Q: What does this do?
I see on L14 that we use `new Date()` which would return a date object.
But `new Date().getDate() - 17` currently returns `1` for me in ye olde DevTools (since today is Aug 18).
So now `createdAt` looks like a Date or Number?

I think above we used something like this:
```js
    const cutOffDate = new Date();
    cutOffDate.setDate(cutOffDate.getDate() - 16);
```

Reply:
in this case it wasnt necessarily consequential since the dates get converted to ticks but otherwise yes, so updated